backend_new/src/controllers/reserved-community-type.controller.ts

```diff
@@ -24,16 +25,16 @@ import { defaultValidationPipeOptions } from '../utilities/default-validation-pi
 import { ReservedCommunityTypeQueryParams } from '../dtos/reserved-community-types/reserved-community-type-query-params.dto';
 import { IdDTO } from '../dtos/shared/id.dto';
 import { SuccessDTO } from '../dtos/shared/success.dto';
+import { PermissionTypeDecorator } from '../decorators/permission-type.decorator';
+import { JwtAuthGuard } from '../guards/jwt.guard';
+import { PermissionGuard } from '../guards/permission.guard';
 
 @Controller('reservedCommunityTypes')
 @ApiTags('reservedCommunityTypes')
 @UsePipes(new ValidationPipe(defaultValidationPipeOptions))
-@ApiExtraModels(
-  ReservedCommunityTypeCreate,
-  ReservedCommunityTypeUpdate,
-  IdDTO,
-  ReservedCommunityTypeQueryParams,
-)
+@ApiExtraModels(ReservedCommunityTypeQueryParams)
```

Comment:
Why wouldn't create here have that explicit guard?

Reply:
Hey @emilyjablonski on ~line 37 of that file we use the `PermissionGuard` to apply permissions on all the endpoints in this controller. As part of our `permission_policy.csv` file we have 

```
p, admin, reservedCommunityType, true, .*
p, jurisdictionAdmin, reservedCommunityType, true, read
p, anonymous, reservedCommunityType, true, read
```

so only admin can create/update/delete these kinds of records, hopefully that helps! but lemme know if that just muddies the water more